static/src/javascripts/projects/common/modules/article/space-filler.ts

```diff
@@ -16,7 +16,7 @@ type SpacefinderItem = {
 type SpacefinderRules = {
 	bodySelector: string;
 	body?: Node;
-	slotSelector: string;
+	slotSelector: string[];
```

Comment:
Might be worth explaining this. Something like
```suggestion
    // array of selectors for direct descendants of bodySelector to be considered as candidates
	slotSelector: string[];
```

Reply:
Good idea